app/controllers/items_controller.rb

```diff
@@ -223,6 +224,20 @@ def collection_ui
     end
   end
 
+  # Draw form for copyright
+  def edit_copyright
+    @change_set = ItemChangeSet.new(@cocina)
+  end
+
+  # save the copyright form
+  def update
+    change_set = ItemChangeSet.new(@cocina)
+    change_set.validate(copyright: params[:item][:copyright])
+    change_set.save
+    reindex
```

Comment:
Should reindex be performed in change_set.save? Seems that others replicating this pattern might easily forget to include.

Reply:
Is there going to be a race condition between the reindex and the redirect?

Reply:
Why is the reindex triggered here instead of DSA?